src/test/resources/google/cloud/compute/v1/compute.proto.baseline

```diff
@@ -23,195 +23,468 @@ option ruby_package = "Google::Cloud::Compute::V1";
 //
 // Messages
 //
+// Use global external addresses for GFE-based external HTTP(S) load balancers in Premium Tier.
+//
+// Use global internal addresses for reserved peering network range.
+//
+// Use regional external addresses for the following resources:
+//
+// - External IP addresses for VM instances - Regional external forwarding rules - Cloud NAT external IP addresses - GFE based LBs in Standard Tier - Network LBs in Premium or Standard Tier - Cloud VPN gateways (both Classic and HA)
```

Comment:
Can we wrap long lines at 100 characters (preserving indentation)?

Reply:
We can, but it can be tricky in case if there is some code snippet or something which we may tear apart by putting line breaks. I would like keeping the scope of this PR limited to transferring comments to the protos. Prettifying the comments can be done separately later. Also, the language-specific clients are processed by language formatters, which, to the best of my knowledge handle the comments as well. So for the actual clients surface and documentation keeping the comments raw should be ok, as they will be prettified by the language formatters (like google-java-formatter for java).

Reply:
OK to doing this in a follow-up. I think we should make the code we generate conform to our style guides. The language formatters should work for GAPICs, but the protos, too, should adhere to [their style guide](https://developers.google.com/protocol-buffers/docs/style#standard_file_formatting). I think we should do this eventually, maybe line-breaking on spaces, etc. But it's not super-high priority.